feat: Support for Cypress 9 (#37)
kamranayub/cypress-browser-permissions: CONTRIBUTING.md

# Contributing

## Browser Internals

To debug or identify the appropriate preferences to set, you can use the special browser URLs.

### Chromium browsers 

Stored in `preferences` object viewable at:

- Edge: [edge://prefs-internals/](edge://prefs-internals/)
- Chrome: [chrome://prefs-internals/](chrome://prefs-internals/)

#### `managed_content_setting_values`

> Stores "managed" permissions. When a Chrome profile is managed by an organization, settings will take effect here.

Author: ```suggestion
Stores "managed" permissions. Settings specified here will take effect when the profile is launched.
```

> 
> #### `default_content_setting_values`
> 
> Stores "default" permissions. When a Chromium profile is "unmanaged", settings will take effect here.

Author: ```suggestion
```
